components/formats-bsd/pom.xml

```diff
@@ -115,18 +115,25 @@
       <!-- NB: We want this, despite warning from dependency:analyze. -->
       <groupId>xalan</groupId>
       <artifactId>serializer</artifactId>
-      <version>2.7.2</version>
+      <version>${xalan.version}</version>
       <scope>runtime</scope>
     </dependency>
     <dependency>
       <!-- NB: We want this, despite warning from dependency:analyze. -->
       <groupId>xalan</groupId>
       <artifactId>xalan</artifactId>
-      <version>2.7.2</version>
+      <version>${xalan.version}</version>
       <scope>runtime</scope>
     </dependency>
 
     <dependency>
+      <groupId>javax.xml.bind</groupId>
+      <artifactId>jaxb-api</artifactId>
```

Comment:
Yes, it's due to the new module system.  The javax.xml classes are no longer available by default with the JRE.  You have two options

1) enable the module with special command-line arguments
2) explicitly use the jar which provides them

(1) is a short-term measure only; the decoupled modules are going to be completely removed with the next major java release (10.04 I think), and it's also painful because you have to run with special command-line arguments.  (2) fixes things to work with all java versions and will always work, so is the best solution to the problem.